apps/builder/app/shared/db/canvas.server.ts

```diff
@@ -4,6 +4,57 @@ import { utils } from "@webstudio-is/project";
 import { loadByProject } from "@webstudio-is/asset-uploader/server";
 import type { AppContext } from "@webstudio-is/trpc-interface/server";
 
+export const loadProductionCanvasData = async (
+  props: {
+    projectId: Project["id"];
+  },
+  context: AppContext
+): Promise<CanvasData[]> => {
+  const pages: CanvasData[] = [];
```

Comment:
maybe its PagesData and not CanvasData if you are calling it pages?

Reply:
or is it not just pages data? then "pages" is wrong name

Reply:
pagesCanvasData should be better, copied naming from SaaS. As here we have CanvasData for all pages